araproc/analysis/standard_reco.py

```diff
@@ -924,3 +926,364 @@ def get_avg_corr_snr(self, wave_packet, excluded_channels = []):
         avg_hpol_corr_snr = np.mean(avg_hpol_corr_snr)
 
         return avg_vpol_corr_snr, avg_hpol_corr_snr  
+
+    def get_arrival_delays_reco(
+        self, reco_results, rf_channel_IDs, excluded_channels, reference_ch, 
+        which_distance, solution
+    ):
+        """
+        Extract the arrival time of the signals from the reconstruction then 
+        calculate the time delays between each channel and the reference channel
+
+        Parameters
+        ----------
+        reco_results : dict
+            Reco results already with the specific reconstruction reqeusted (so
+            the keys of this object include 'theta' and 'phi')
+        rf_channel_IDs : list
```

Comment:
What's the reason for this argument? We don't just wanna pull from `const.rf_channels_ids`?

Reply:
I did not know that that existed haha. Updated!